core/src/debug/assets/mobile-core.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
@secondsun Let not commit that here. 
We may just have that as part of unit tests and write unit tests?

I think that initial name suggested was  `mobile-services.json`, but I think that best will be to not hardcode it.
We can have constant or build time variable for that.

Reply:
Yes, this file is just for unit tests.

In Android I had problems getting the unit test to pick up the asset unless it was in the debug source set.  This seems to be a common work around and, since it didn't get bundled in the final artifact, I didn't wsee a problem.